lib/manageiq/messaging/kafka/common.rb

```diff
@@ -21,12 +21,9 @@ def consumer(beginning, options)
           @consumer = kafka_client.consumer
         end
 
-        def raw_publish(wait, body, options)
```

Comment:
I didn't like how `raw_publish(true, ...` and `raw_publish(false, ...` looked as the caller, it wasn't clear what the first arg was doing.  This refactoring is in its own commit if we want to split it out into a separate PR if this is too much change.

Reply:
I like removing the first argument but it feels like a good concept. Can we introduce 2 different methods? Something like `raw_publish_sync` and `raw_publish_blocking`, or call one `raw_publish` and the other with the longer name?

EDIT: if `raw_publish` is an internal method and you moved the `wait` into the caller, then this is very nice.

Reply:
> EDIT: if raw_publish is an internal method and you moved the wait into the caller, then this is very nice.

Yes this is an internal method only